docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,404 @@
+```javascript
+  import { getInitialTestAccountsWallets } from "@aztec/accounts/testing";
+  //...
+    testWallets = await getInitialTestAccountsWallets(pxe);
+```
+
+Note: The test account addresses can also be seen in the sandbox logs when it starts up.
+
+**Create and deploy a new account, paid for by another account**
+
+For the sandbox only, you can use the test accounts provided at aliases: `test0`, `test1`, `test2`
+
+```bash
+aztec-wallet create-account -a alice --payment method=fee_juice,feePayer=test0
+```
+
+:::warning
+Specifying the `feePayer` is only an option for commands that deploy an account.
+:::
+
+The equivalent using aztec.js - Create a fee juice payment method with a test account as sender, then deploy an account:
+
+```javascript
+import { FeeJuicePaymentMethod } from "@aztec/aztec.js";
+  //... building upon the previous section
+  const useFeeJuice = new FeeJuicePaymentMethod(testWallets[0].getAddress())
+  await schnorrAccount.deploy({ fee: { paymentMethod: useFeeJuice }}).wait()
+```
+
+:::note Payment: Sponsored Fee Paying Contract
+Options for explicitly stating fee_juice from the sender which is the default payment method:
+- CLI: `--payment method=fee_juice` (default)
+- .js: `{ fee: { paymentMethod: new FeeJuicePaymentMethod(<fee payer address>) }`
+:::
+
+### Bridging Fee Juice
+
+The sandbox allows free-minting on it's L1 to be bridged and claimed on its Aztec node.
+For testnet you'll first need an L1 account with the fee asset to send the fees to the Aztec bridge.
+
+We'll register a new account `accBFJ` and bridge fee-juice to it.
+
+```bash
+aztec-wallet create-account -a accBFJ --register-only
+aztec-wallet bridge-fee-juice 1000000000000000000 accBFJ --mint --no-wait
+```
+
+You'll have to wait for two blocks to pass for bridged fee juice to be ready on Aztec. For the sandbox you can do this by putting through two arbitrary transactions. Eg:
+```bash
+aztec-wallet deploy counter_contract@Counter --init initialize --args 0 accounts:test0 --from test0 -a counter
+aztec-wallet send increment -ca counter --args accounts:test0 accounts:test0 --from test0
+```
+
+Now the funded account can deploy itself with the bridged fees, claiming the bridged fee juice and deploying the contract in one transaction:
+
+```bash
+aztec-wallet deploy-account --from accBFJ --payment method=fee_juice,claim
+```
+
+
+The equivalent using aztec.js - bridge fee juice, (pass two txs), create and use payment method:
+
+(See [here](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/cli-wallet/src/cmds/bridge_fee_juice.ts#L32) to initialise a fee juice portal manager)
+
+```javascript
+import { FeeJuicePaymentMethod, PrivateFeePaymentMethod, PublicFeePaymentMethod } from "@aztec/aztec.js";
+  const newWallet = await schnorrAccount.getWallet()
+  const feeJuiceReceipient = schnorrAccount.getAddress()
+
+  const claim = await feeJuicePortalManager.bridgeTokensPublic(feeJuiceReceipient, 1000000000000000000n, true);
```

Comment:
How do you set up `feeJuicePortalManager`? This is the first time you use it.

Reply:
you can see how its done here https://github.com/AztecProtocol/aztec-starter/blob/dd2b385ab92758343e8ff9a7b1dcca29e7c8c8c0/scripts/fees.ts#L69

Reply:
Above this I link to code that shows an implementation using @aztec/etherem, I can instead bring that code in here, but it makes the code snippet much larger than the bite-size chunks I had aimed for.

The code linked to is from aztec-wallet (bridgeL1FeeJuice command in bridge_fee_juice.ts), and was deliberately chosen to communicate the .ts inside `aztec-wallet` is a source for other things they may need to do.